articles/language/expressions.md

```diff
@@ -127,10 +127,10 @@ The two `Bool` literal values are `true` and `false`.
 
 Given any two expressions of the same primitive type, the `==`
 and `!=` binary operators may be used to construct a `Bool` expression.
-The expression will be true if the two expressions are (resp. are not) equal.
+The expression will be true if the two expressions are equal, and false if not.
 
-Values of user-defined types may not be compared, only their values can be compared. 
-For example,
+Values of user-defined types may not be compared, only their unwrapped values can be compared. 
```

Comment:
The slight confusion that might arise here is if the unwrapped values can't be compared; for instance, if considering `newtype Foo = Int; newtype Bar = Foo;` you'd need `bar1!! == bar2!!`. While that's probably not often the best solution, I wouldn't want users to misunderstand what's happening in the double-wrap case either.